.vscode/launch.json

```diff
@@ -9,13 +9,13 @@
             "name": "Listen for XDebug",
             "type": "php",
             "request": "launch",
-            "port": 9000
+            "port": 9003
```

Comment:
The new XDebug port configuration does not match the documented port in the project's README. Please do either document the new port value in the project's README or undo this change to preserve the old port value. I would prefer the second possibility to avoid misconfigurations in already existing development setups.